shared-functions

```diff
@@ -135,9 +135,17 @@ write_uptimer_deploy_config() {
   local available_port
   available_port=${AVAILABLE_PORT:-"-1"}
 
+  if [ -z "${USE_SINGLE_APP_INSTANCE}" ]; then
+    USE_SINGLE_APP_INSTANCE=false
+  fi
   set +x
   local cf_admin_password
-  cf_admin_password=$(get_password_from_credhub cf_admin_password)
+
+  if [ -z "${CF_ADMIN_PASSWORD}" ]; then
+     cf_admin_password=$(get_password_from_credhub cf_admin_password)
+  else
+    cf_admin_password=$(echo ${CF_ADMIN_PASSWORD})
+  fi
 
```

Comment:
can you elaborate on why that's the case ? The @cloudfoundry/cf-diego team will be switching our environments to use credhub soon but in the meantime we would like to use the vars-store.

Reply:
Primarily, this change doesn't belong in the pull request because it's unrelated to the `use_single_app_instance` parameter.

@cloudfoundry/cf-release-integration recently made the decision to move cf-deployment and cf-deployment-concourse-tasks to Credhub. These braking changes led to the 7.0 release of cf-deployment-concourse-tasks. If we were to add `CF_ADMIN_PASSWORD` to this task, removing it would require another major version bump.

If you guys would like help migrating to Credhub, we'd be more than happy to cross team pair 
sometime soon to get you on Credhub.